tests/test_fork_provider.py

```diff
@@ -54,11 +44,11 @@ def contract_container(contract_type) -> ContractContainer:
     return ContractContainer(contract_type=contract_type)
 
 
-@pytest.fixture()
+@pytest.fixture(scope="module")
```

Comment:
All these should be module scoped, probably fixes the alchemy issue right here tbh

Reply:
ok great, I was trying to remove the scope because I wasn't sure what should be scoped and what shouldn't be and was just trying things to see what stuck.